validation/src/main/scala/hmda/validation/ValidationStats.scala

```diff
@@ -0,0 +1,65 @@
+package hmda.validation
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import hmda.model.fi.SubmissionId
+import hmda.persistence.messages.CommonMessages.{ Command, Event, GetState }
+import hmda.persistence.model.HmdaPersistentActor
+
+object ValidationStats {
+
+  def name = "ValidationStats"
+  case class SubmissionStats(id: SubmissionId, totalLars: Int)
+  case class AddSubmissionStats(stats: SubmissionStats) extends Command
+  case class SubmissionStatsAdded(stats: SubmissionStats) extends Event
+  case class FindTotalLars(institutionId: String, period: String) extends Command
+
+  def props(): Props = Props(new ValidationStats)
+
+  def createValidationStats(system: ActorSystem): ActorRef = {
+    system.actorOf(ValidationStats.props(), "validation-stats")
+  }
+
+  case class ValidationStatsState(stats: Seq[SubmissionStats] = Nil) {
+    def updated(event: Event): ValidationStatsState = event match {
+      case SubmissionStatsAdded(s) =>
+        ValidationStatsState(stats :+ s)
+    }
+  }
+}
+
+class ValidationStats extends HmdaPersistentActor {
+  import ValidationStats._
+
+  override def preStart(): Unit = {
```

Comment:
No, I left that out on purpose. The bug we were seeing was related to `super.preStart()`, where the `HmdaPersistentActor` uses a `receive timeout` to shut itself down. This 'ValidationStats' actor is a top level actor that is supposed to be running all the time, so there is no need to shut it down after a period of inactivity